WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m

```diff
@@ -766,32 +766,40 @@ - (UITableViewCell *)configureFeaturedImageCellForIndexPath:(NSIndexPath *)index
 - (UITableViewCell *)configureShareCellForIndexPath:(NSIndexPath *)indexPath
 {
     UITableViewCell *cell;
+    BOOL canEditSharing = ![self.apost.status isEqualToString:PostStatusPublish];
```

Comment:
For this, we actually use these statuses as more of what status the post _will_ have versus what it currently has. For example, since this change, creating a new post and going to post options renders the Publicize sections disabled since the post is actually unpublished, but carries the publish status.